model/src/test/kotlin/AdvisorRecordTest.kt

```diff
@@ -84,7 +85,7 @@ class AdvisorRecordTest : WordSpec({
     }
 
     "hasIssues" should {
-        "detect a result that has no issues" {
+        "return false given the record has no issues" {
```

Comment:
Nit: Strictly speaking that's an unrelated change that should go to a separate commit. But let's turn a blind eye on that for now.